pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -468,6 +468,16 @@ def test_groupby_agg_coercing_bools():
     tm.assert_series_equal(result, expected)
 
 
+def test_groupby_agg_nested_dictionary():
```

Comment:
You're not testing nested dictionaries here, so I think this name is misleading. I think we test dictionary behavior elsewhere, what seems likely to be novel about this test is that you are using ``[["B"]]``; this calls the function `_gotitem` in the groupby code. So I'd suggest a name like: `test_groupby_agg_dict_with_gotitem`

Reply:
getitem (as in the dunder `__getitem__`) would also work

Reply:
Done!